services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/SwitchFailuresSpec.groovy

```diff
@@ -1,4 +1,4 @@
-package org.openkilda.functionaltests.spec
+package org.openkilda.functionaltests.spec.isl
```

Comment:
we already have 'spec.links' package. I find it pretty much the same meaning as 'isl'. Can we leave either isls or links?

Reply:
Let's have a separate discussion about this offline.